src/libpriv/rpmostree-postprocess.cxx

```diff
@@ -205,19 +198,12 @@ process_kernel_and_initramfs (int rootfs_dfd, JsonObject *treefile, gboolean uni
 
   /* Run dracut with our chosen arguments (commonly at least --no-hostonly) */
   g_autoptr (GPtrArray) dracut_argv = g_ptr_array_new ();
-  if (treefile && json_object_has_member (treefile, "initramfs-args"))
+  rust::Vec<rust::String> initramfs_args;
+  if (treefile)
     {
-      JsonArray *initramfs_args = json_object_get_array_member (treefile, "initramfs-args");
-      guint len = json_array_get_length (initramfs_args);
-
-      for (guint i = 0; i < len; i++)
-        {
-          const char *arg
-              = _rpmostree_jsonutil_array_require_string_element (initramfs_args, i, error);
-          if (!arg)
-            return FALSE;
-          g_ptr_array_add (dracut_argv, (char *)arg);
-        }
+      initramfs_args = treefile->get ().get_initramfs_args ();
```

Comment:
OK well, that's just evil.  There's *two* initramfs args, one for derived configs and one for base composes, and this gets the former.